src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -177,6 +177,7 @@ export default {
 		box-sizing: border-box;
 		display: flex;
 		flex-direction: column;
+		gap: 4px;
```

Comment:
The variable `--default-grid-baseline` was added by @juliushaertl, it can be used here. :)

Reply:
Done